_component_design/popovers.md

```diff
@@ -14,81 +14,171 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-Popovers are triggered as a result of a user action and provide a user with
-additional context through language, actions through buttons, or both. Popovers
-come in a the following types Hover Popover, Triggered Popover, and Closeable
-Popover
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
 
-# Hover Popover
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to Use
+{: .hxSubSectionTitle}
+Popovers are triggered as a result of a user action and provide a user with additional context through language, actions through buttons, or both. 
+
+Popovers may also be used for short create/edit workflows, and are particularly useful if context from the page is needed while making the change. If a large workflow is required, consider creating a new page.
```

Comment:
Editorial: Remove comma in the first sentence.

Reply:
removed.